contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -129,7 +129,7 @@ contract DisputableAragonApp is IDisputable, AragonApp {
     */
     function _newAgreementAction(uint256 _disputableActionId, bytes _context, address _submitter) internal returns (uint256) {
         IAgreement agreement = _ensureAgreement();
-        return agreement.newAction(_disputableActionId, _context, _submitter);
+        return agreement.newAction.value(msg.value)(_disputableActionId, _context, _submitter);
```

Comment:
I wonder if we shouldn't expose this as a parameter instead; maybe the app takes a cut of ETH itself or etc.

Reply:
If the app manipulates it, the agreement app won't be able to pay fees, unless the user is sending more than the required amount. However, how would a parameter help here?

Reply:
Yes, I was imagining the user sending more fees to the app (because of app-specific logic).

As an example, in the case of a registry, maybe you need to stake some additional ETH into the app as well as pay the fee.

The parameter would help here because the app then has control over how much value is sent.